plotsearch/views/plot_search.py

```diff
@@ -383,13 +383,6 @@ class AreaSearchAttachmentViewset(
         "DELETE": ["plotsearch.delete_areasearchattachment"],
     }
 
-    def get_queryset(self):
-        qs = super().get_queryset()
-        user = self.request.user
-        if user.has_perm("plot_search.area_search_attachment"):
```

Comment:
Based on this code, if the user has this permission, the user would be able to find the file.
If the user does not have the permission, they can view only their own uploaded files.

What is this pull request fixing?